docs/development/build-instructions-gn.md

```diff
@@ -90,29 +90,35 @@ $ gclient sync -f
 
 ## Building
 
+There are three different build configurations you can potentially set up - `Debug`, `Testing`, and `Release`.
+
+In your setup, you should replace `OUT_DIR` with your desired build configuration of either `Debug`, `Testing`, and `Release`, and `out_dir.gn` correspondingly with `debug.gn`, `testing.gn` or `release.gn`.
+
+On Unix-based machines:
 ```sh
 $ cd src
 $ export CHROMIUM_BUILDTOOLS_PATH=`pwd`/buildtools
 # this next line is needed only if building with sccache
 $ export GN_EXTRA_ARGS="${GN_EXTRA_ARGS} cc_wrapper=\"${PWD}/electron/external_binaries/sccache\""
-$ gn gen out/Debug --args="import(\"//electron/build/args/debug.gn\") $GN_EXTRA_ARGS"
+$ gn gen out/OUT_DIR --args="import(\"//electron/build/args/out_dir.gn\") $GN_EXTRA_ARGS"
```

Comment:
I kinda liked being able to just copy-paste these.  Is there a way we can retain the COPY_PASTE nature? maybe using an env var?

Reply:
i don't disagree - what i've noticed though is that people will copy paste these things and not understand why they're doing that, so no one ends up using the testing build even though that makes more sense in a lot of cases 🤔